src/widgets/dialogs/SelectChannelDialog.cpp

```diff
@@ -404,10 +426,12 @@ bool SelectChannelDialog::EventFilter::eventFilter(QObject *watched,
                 this->dialog->ui_.twitch.whispers->setFocus();
                 return true;
             }
-            else
+            else if (widget == this->dialog->ui_.twitch.live)
             {
-                widget->nextInFocusChain()->setFocus();
```

Comment:
why was this removed?

Reply:
It's not removed...
It's on line 434 now.
I added logic for going from the bottom element to the top one.